src/engine/source/indexerconnector/src/indexerConnector.cpp

```diff
@@ -158,22 +158,25 @@ IndexerConnector::IndexerConnector(const IndexerConnectorOptions& indexerConnect
             {
                 // Process data.
                 HTTPRequest::instance().post(
-                    {HttpURL(url), bulkData, secureCommunication},
-                    {[functionName = logging::getLambdaName(__FUNCTION__, "handleSuccessfulPostResponse")](
-                         const std::string& response)
+                    {.url = HttpURL(url), .data = bulkData, .secureCommunication = secureCommunication},
+                    {.onSuccess = [functionName = logging::getLambdaName(__FUNCTION__, "handleSuccessfulPostResponse")](
+                                      const std::string& response)
                      { LOG_DEBUG_L(functionName.c_str(), "Response: %s", response.c_str()); },
```

Comment:
improve logs: this will not log correctly. spdlog uses fmt::format, please correct these logs

Reply:
That's the information I needed, it was not clear to me in the description, thanks. On it.